include/boost/geometry/algorithms/detail/buffer/buffer_inserter.hpp

```diff
@@ -537,7 +537,7 @@ struct buffer_inserter_ring
 
         if (n >= min_points)
         {
-            detail::normalized_view<RingInput const> view(simplified);
+            detail::reverse_close_view<RingInput const> view(simplified);
```

Comment:
Why is this a `reverse` view? This looks wrong to me. But maybe it's the name changes I don't fully understand yet.

Reply:
I believe I get it now. It is a view that **can** reverse and it **can** close, both if necessary.

In the past this was the definition of **normalized** 

So it should be correct, but I actually prefer the old name (or another name)

Reply:
This naming is taken from the C++20 convention. Have a look at this: https://en.cppreference.com/w/cpp/ranges
e.g.: `drop_view`, `take_view`, `transform_view`, `split_view`, `join_view`, `reverse_view`. AFAIU the names are the same as corresponding algorithms/functions, i.e. do X with this view. So `reverse_close_view` is short for `reverse_and_close_view`.

Boost.Range have been using a different convention naming views by what a range becomes after applying a view, e.g. `reversed_view` is a range that was reversed. Using this convention it'd be `reversed_and_closed_view` or `reversed_closed_view`.

I'm open to suggestion regarding the naming but I think that `normalized_view` is a wrong name because normalization may mean different things. So I'd prefer to name the views to mean precisely what they are doing or becoming, whatever this is.

Reply:
I'd like to stress one more thing. The difference between these views and C++20 ones is that these are done conditionally unlike e.g. `std::ranges::reverse_view` which always reverses a range, just like `boost::reversed_view` or `std::reverse` or in fact `boost::geometry::reverse()`. This is why I added `conditionally` in the alternatives.

Reply:
Thanks for your explanations. It's clear and I agree. Indeed, normalization is ambiguous.
`Conditionally` is OK, a bit long though.